website/src/Components/PageComponents/AccountCreationForm.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+import { FormGroup, ControlLabel, FormControl, HelpBlock, Col, Button, Panel } from 'react-bootstrap';
+import '../../resources/index.css';
+
+class AccountCreationForm extends Component{
+	constructor(props, context) {
+		super(props, context);
+		this.handleChange = this.handleChange.bind(this);
+		this.state = {
+				email: '',
+				password: '',
+				confirmPassword: '',
+				firstName: '',
+				lastName: '',
+				emailValid: false,
+				passwordValid: false,
+				confirmPasswordValid: false
+		};
+
+		this.onSubmit.bind;
+	}
+
+	handleChange(e) {
+    	this.setState({ [e.target.id]: e.target.value });
+  	}
+
+  	getEmailValidationState(){
+  		if(this.state.email.indexOf('@') > -1){
```

Comment:
You can use this line of code to more accurately check if text follows an email pattern
```
let pattern = /^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
if (this.state.email == "") {
    return 'error';
} else if (!pattern.test(this.state.email)) {
   return 'error';
} else {
    this.setState({emailValid: true});
    return 'success';
}
```